jwql/website/apps/jwql/data_containers.py

```diff
@@ -266,18 +268,28 @@ def get_edb_components(request):
 
                 mnemonic_exploration_result.n_rows = len(mnemonic_exploration_result)
 
+                # generate tables for display and download in web app
                 display_table = copy.deepcopy(mnemonic_exploration_result)
                 # temporary html file, see http://docs.astropy.org/en/stable/_modules/astropy/table/
-                # table.html#Table.show_in_browser
                 tmpdir = tempfile.mkdtemp()
-                path = os.path.join(tmpdir, 'mnemonic_exploration_result_table.html')
-                with open(path, 'w') as tmp:
+                file_name_root = 'mnemonic_exploration_result_table'
+                path_for_html = os.path.join(tmpdir, '{}.html'.format(file_name_root))
+                with open(path_for_html, 'w') as tmp:
                     display_table.write(tmp, format='jsviewer')
-                mnemonic_exploration_result.html_file = path
-                mnemonic_exploration_result.html_file_content = open(path, 'r').read()
+                mnemonic_exploration_result.html_file_content = open(path_for_html, 'r').read()
+
                 # pass on meta data to have access to total number of mnemonics
                 mnemonic_exploration_result.meta = meta
 
+                # save file locally to be available for download
+                STATIC_DIR = os.path.join(settings.BASE_DIR, 'static')
```

Comment:
@Johannes-Sahlmann I don't think this is actually used as a global variable?  And so I think it could be named with lowercase instead of uppercase per PEP8.

Reply:
thanks. I changed it to lower case.